src/bare_metal_billing/models.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass, field
+from typing import Annotated
+from typing_extensions import Self
+import datetime
+
+import pydantic
+
+
+def parse_date(v: str | datetime.date) -> datetime.date:
+    if isinstance(v, str):
+        return datetime.datetime.fromisoformat(v)
+    return v
+
+
+def parse_bm_accelerator(v: list) -> int:
+    return len(v)
+
+
+PositiveInt = Annotated[int, pydantic.Field(gt=0)]
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Thank you. I forgot to used this in my model :P